src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx

```diff
@@ -1,98 +1,23 @@
-import {__} from '@wordpress/i18n';
-import FieldSettings from '../settings/Edit';
-import {useFieldNameValidator} from '../../../hooks';
-import {InspectorAdvancedControls, InspectorControls} from '@wordpress/block-editor';
-import {ExternalLink, PanelBody, PanelRow, TextControl, ToggleControl} from '@wordpress/components';
-import {slugify} from '@givewp/form-builder/common';
-import {useCallback} from '@wordpress/element';
+import {TextControl} from '@wordpress/components';
 import {BlockEditProps} from '@wordpress/blocks';
 
-export default function Edit({attributes, setAttributes}: BlockEditProps<any>) {
-    const {fieldName, label, storeAsDonorMeta, displayInAdmin, displayInReceipt} = attributes;
-    const validateFieldName = useFieldNameValidator();
-
-    const updateFieldName = useCallback(
-        (newFieldName) => {
-            setAttributes({
-                fieldName: slugify(newFieldName),
-            });
-        },
-        [setAttributes]
-    );
-
-    const enforceUniqueFieldName = useCallback(() => {
-        const [isUnique, suggestedName] = validateFieldName(fieldName, '');
-        if (!isUnique) {
-            updateFieldName(suggestedName);
-        }
-    }, [fieldName, updateFieldName, validateFieldName]);
-
-    const enforceRequiredValue = useCallback(() => {
-        if (!fieldName) {
-            updateFieldName(label);
-        }
-    }, [fieldName, updateFieldName, label]);
```

Comment:
It doesn't look like this was carried over properly

Reply:
Wow! Strange that didn't throw an error! Good catch! Resolved in 154e07c